src/lib/Scenes/Artwork/Components/CommercialButtons/__tests__/CommercialButtons-tests.tsx

```diff
@@ -43,76 +34,78 @@ const componentWithQuery = async ({ mockArtworkData, mockOrderMutationResults, m
   })
 }
 
+const relayComponent = async ({ artwork }) => {
```

Comment:
would it make sense to add a test for the bid button in here?

Reply:
Added!